packages/cli/src/commands/init-flow.ts

```diff
@@ -107,9 +109,30 @@ export const initFlow = async (
       "Now you can:",
       folderName && `Go to your project: ${pc.dim(`cd ${folderName}`)}`,
       `Run ${pc.dim("npm run dev")} to preview your site on a local server.`,
-      `Run ${pc.dim("npx vercel")} to publish on Vercel.`,
+      projectTemplate && getDeploymentInstructions(projectTemplate),
     ]
       .filter(Boolean)
       .join("\n")
   );
 };
+
+const getDeploymentInstructions = (
```

Comment:
why making it a function vs just a config?

Reply:
We have things like `netlify-functions` and `netlify-edge-functions` with same instructions. So, switch can make it easy to organise things around.

Reply:
got it